### PR TITLE
Implement Pause Menu

### LINKED ISSUE #59: Pause Menu Design
- Resume;
- Exit;

### LINKED ISSUE #60: Implement Design: Pause Menu
Linked: https://github.com/FlamingoFiestaStudio/OlympusGoneWild/issues/59

### PR DESCRIPTION
**Pull request by Adan0k**
### Image/Video
image

### Related Issue

Closes #60 

### Does this PR introduce a breaking change? ⚠️

- [x] No
- [ ] Yes 

### Tests

- [ ] I tested all game E2E;
- [x] I just tested some scenes;
- [ ] Not applicable.

### Check list

Before submitting this pull request, please ensure the following are completed:

- [x] [My commits follow the project conventions](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#commit-message-convections).
- [x] My code follows the project's coding style and conventions.
- [ ] I have updated the project documentation (if necessary).
- [x] My code makes sense with the design.

## Additional Notes

---

Thank you for your contribution to Olympus Gone Wild! Your effort is greatly appreciated, and it helps make this project better for everyone. 😄🎮🚀

### CODE REVIEW

**Review by Gabriel-Sous-a — APPROVED** [review]

**Comment by Gabriel-Sous-a** [comment]
When clicked on `Resume` the player shoots, this should be fixed

**Review by 201flaviosilva — COMMENTED** [review]
Nice work 👌

**Review by Goncalo-FP — COMMENTED** [review]
When clicked on `Resume` the player shooting is not resolved.

**Comment by Adan0k** [comment]
> When clicked on `Resume` the player shooting is not resolved.

out of scope

**Comment by Gabriel-Sous-a** [comment]
you have conflicts to solve and need to update the branch, you're already 8 commits behind

**Review by Goncalo-FP — APPROVED** [review]

File: menus/assets/pause/title.png
Comment (201flaviosilva) [inline menus/assets/pause/title.png:1]: Remove this file :)

File: menus/assets/pause/resume.png
Comment (201flaviosilva) [inline menus/assets/pause/resume.png:1]: Remove this file :)

File: menus/assets/pause/quit.png
Comment (201flaviosilva) [inline menus/assets/pause/quit.png:1]: Remove this file :)

File: menus/assets/pause/options.png
[the remark this answers is not in the document]
Comment (201flaviosilva) [reply menus/assets/pause/options.png:1]: Remove this file :)

File: menus/assets/typography/PressStart2P-Regular.ttf
Comment (201flaviosilva) [inline menus/assets/typography/PressStart2P-Regular.ttf:1]: Pls give credits to the original author of this font

File: menus/pause_menu.gd [hunk 1]
```diff
@@ -17,9 +17,11 @@ func _input(event : InputEvent):
 	if(event.is_action_pressed("ui_cancel")):
 		game_paused = !game_paused
 
-func _on_resume_button_pressed():
+func _on_resume_btn_pressed():
+	await get_tree().create_timer(0.1).timeout
```
Comment (201flaviosilva) [inline menus/pause_menu.gd:21]: ????

**Review by 201flaviosilva — COMMENTED** [review]
:)

**Comment by 201flaviosilva** [comment]
This pull request was closed because the project was archived

**Pull request closed without merging**